README.md

```diff
@@ -216,6 +216,7 @@ Table of Contents
   * [Dataimporter.io](https://www.dataimporter.io) - Tool for connecting, cleaning, and importing data into Salesforce. Free Plan includes up to 20,000 records per month.
   * [Data Miner](https://dataminer.io/) - A browser extension (Google Chrome, MS Edge) for data extraction from web pages CSV or Excel. The free plan gives you 500 pages/month.
   * [Datapane](https://datapane.com) - API for building interactive reports in Python and deploying Python scripts and Jupyter Notebooks as self-service tools.
+  * [DB-IP](https://db-ip.com/) - Free IP geolocation API with 1k request per IP per day.lite database under CC-BY 4.0 License is free too.
```

Comment:
Please share a link where it says about these free queries, I cant find that on the pricing section.

thanks

Reply:
This page [https://db-ip.com/api/free](https://db-ip.com/api/free)

Reply:
Please upadate the link to poijnt to that then, its not really discoverable from the front page.